frontend/src/SideBarSearch/searchCard.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import Card from '@material-ui/core/Card';
+import CardMedia from '@material-ui/core/CardMedia';
+import { makeStyles, createStyles, Theme } from "@material-ui/core/styles";
+import Typography from '@material-ui/core/Typography';
+
+const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+        flexContainer: {
+            display: "flex",
+            flexDirection: "column",
+            alignItems: "flexStart",
+        },
+        card: {
+            width: "450px",
```

Comment:
width should be proportional (%) for various screen sizes

Reply:
the thing is if I were to put a percentage the sidebar would take the whole screen. 
![Screen Shot 2021-01-26 at 9 35 16 PM](https://user-images.githubusercontent.com/52011886/105934534-62ee4580-601e-11eb-8035-28690ca4dc77.png)